tests/core/test_decorators.py

```diff
@@ -11,38 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import pytest
-import torch
 
-from pytorch_lightning.core.decorators import auto_move_data
-from tests.helpers import BoringModel
-from tests.helpers.runif import RunIf
 
-
-@RunIf(min_gpus=1)
-@pytest.mark.parametrize(
-    ["src_device", "dest_device"],
-    [
-        pytest.param(torch.device("cpu"), torch.device("cpu")),
-        pytest.param(torch.device("cpu", 0), torch.device("cuda", 0)),
-        pytest.param(torch.device("cuda", 0), torch.device("cpu")),
-        pytest.param(torch.device("cuda", 0), torch.device("cuda", 0)),
-    ],
-)
-def test_auto_move_data(src_device, dest_device):
-    """Test that the decorator moves the data to the device the model is on."""
-
-    class CurrentModel(BoringModel):
-        @auto_move_data
-        def forward(self, *args, **kwargs):
-            return super().forward(*args, **kwargs)
-
-    model = CurrentModel()
-    model = model.to(dest_device)
-    model.prepare_data()
-    loader = model.train_dataloader()
-    x = next(iter(loader))
-
-    # test that data on source device gets moved to destination device
-    x = x.to(src_device)
-    assert model(x).device == dest_device, "Automoving data to same device as model failed"
+def test_parameter_validation():
+    # TODO: Fill out this test
+    assert True
```

Comment:
This file is empty otherwise so I added a stub for the other untested decorator rather than deleting the file.

Reply:
I think it's okay to delete the file, unless @kaushikb11 has in mind a possible test for it

Reply:
I'll remove for now and the file can be reinstated later